src/api/gnosisProtocol/api.ts

```diff
@@ -112,7 +112,7 @@ function _mapNewToLegacyParams(params: FeeQuoteParams): OrderQuoteRequest {
     appData: getAppData().appDataKeccak256,
     validTo,
     partiallyFillable: false,
-    priceQuality: priceQuality ? (priceQuality as PriceQuality) : undefined,
+    priceQuality: priceQuality as PriceQuality,
```

Comment:
it was there i didn't add it :P , its cause price quote was a mess!
we need to use the enum instead of string as it is right now. Also waiting for the SDK to do this small improvement

Reply:
Same as all the other comments. solved once i have the new SDK

https://github.com/cowprotocol/cowswap/pull/2945/commits/6d61111fc0f0e7696e19b53418a3a7b0653c52ae